gadfly/gap_fill_generator.py

```diff
@@ -30,15 +30,28 @@ def find_named_entities(self):
         return entities
 
     def summarize_sentences(self):
-        fs = FrequencySummarizer()
-        sents = []
-        for span in self._parsed_text.sents:
-            sent = [self._parsed_text[i] for i in range(span.start, span.end)]
-            tokens = []
-            for token in sent:
-                tokens.append(token.text)
-            sents.append(tokens)
-        sentences = fs.summarize(sents, 5)
+        option = 1
```

Comment:
Not sure what the intention is here. What is option? If it is a switch between the two different types of summarizers, we would be better off calling the summarizer explicitly.

Switch statements are bad. I can't find the exact article that I want but this gets pretty close to it: https://schneide.wordpress.com/2010/12/13/avoid-switch-use-enum/.